playwright-e2e/authentication/auth-base.ts

```diff
@@ -0,0 +1,44 @@
+import { Page } from '@playwright/test';
+import { generateEmailAlias } from 'utils/faker-utils';
+
+/**
+ *
+ * @param {Page} page
+ * @param {{email: string, password: string, waitForNavigation?: boolean}} opts
+ * @returns {Promise<void>}
+ */
+export async function fillInEmailPassword(
```

Comment:
Create new class `auth-base.ts` to include common functions to all `auth-*` classes to eliminate boilerplate code.